docs: remove the sponsors and update the READme (#1770)
asyncapi/community: README.md

 
 [![AsyncAPI Logo](./assets/github-repobanner-community.png)](https://www.asyncapi.com)

# 📘 AsyncAPI Community Guide

Welcome to the AsyncAPI Initiative! Our mission is to improve the current state of Event-Driven Architectures (EDAs) and the tooling ecosystem around them.
Join us in shaping the future of how you work with Event-Driven Architectures.

## Table of Contents

- [Community Goals](#community-goals-🎯)
- [Contributing to Async API Initiative](#contributing-to-asyncapi-initiative-❤️)
  - [Getting Started](#getting-started)
  - [Code of Conduct](#code-of-conduct-🔖)
- [Contributing to Async API Docs](#how-to-contribute-to-our-docs-🛠)
- [Our Charter](#our-charter-📜)
- [Community Governance](#community-governance-🏛️)
- [Technical Steering Committee (TSC)](#async-api-technical-steering-committee-tsc-👥)
    - [How to Be a part of TSC](#how-to-be-a-part-of-tsc-🤝)
    - [Meet AsyncAPI's TSC](#meet-async-apis-current-tsc-members)
- [Ambassador Program](#asyncapi-ambassador-program-🌟)
- [Community Discussions](#community-discussions-💬)

---

## Community Goals 🎯

> Join us in 2025 as we boost maintainers, streamline onboarding, and create a structured Community Working Group Umbrella. Let's grow together through exciting initiatives like the AsyncAPI Grant Program, conferences, and mentorship. <br>

Reviewer: This section changes and relies on the community goals that change each year. Rather, make it dynamic

Be a part of our vibrant, evolving community!

**[More details »](./annual-goals/2025_Community_Goals.md)**

---

## Contributing to AsyncAPI Initiative ❤️
#### We love your input ! Contributing is easy and transparent.

   - ### Getting Started
      - **Spec Contribution:** [AsyncAPI Spec Guide](https://github.com/asyncapi/spec/blob/master/CONTRIBUTING.md)

>       - **Tools Contribution:** [AsyncAPI Tools Guide](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md)

Reviewer 2: Do we wanna include only Spec and Tools contribution here? @thulieblack

Reviewer: No this section we should remove

   - ### Code of Conduct 🔖
      - Read our [Code of Conduct](./CODE_OF_CONDUCT.md) to understand our community values and expected behavior.
   
#### Follow these steps to make your contribution count!

```markdown
┌───────────────────────────────┐
│    Open an issue 🐞💡         │
│    (Bug report or feature)    │
└───────────────────────────────┘
               ⇩
┌───────────────────────────────┐
│    Open a Pull Request 🚀     │
│    (After issue approval)     │
└───────────────────────────────┘
               ⇩
┌───────────────────────────────┐
│ Your changes will be merged 🎉│
│   and released soon 🚢        │
└───────────────────────────────┘
```
---

> ## Community Documentation 📚

Reviewer: This section should instead be part of the contributing section

Welcome to the Community Documentation directory! Our goal is to collaboratively maintain a collection of helpful community-related documentation, from becoming an AsyncAPI contributor to becoming a TSC member, and beyond.

  ### How to Contribute to Our Docs 🛠

  We believe that **code is cool**, but **great docs make all the difference**! Find more about contributing to our Docs at **[AsyncAPI Docs](https://www.asyncapi.com/docs/community)**.

  ### Getting Started 🚀

1. **Familiarize Yourself**: Review our [Contribution Guide](https://github.com/asyncapi/community/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).
2. **Check the Docs Board**: Visit our [AsyncAPI Docs Board](https://github.com/orgs/asyncapi/projects/12).
3. **Pick an Issue**: Choose an issue to work on and leave a comment introducing yourself. Feel free to ask questions if you're unsure how to get started.
4. **Open a PR**: If no work has been done on the chosen issue, open a PR and start contributing!

> ### Technical Writer Onboarding ✍️

Reviewer: Put everything related to onboarding under one section

- Review the list of [technical writer contributor responsibilities](https://www.asyncapi.com/docs/community/onboarding-guide/technical-writer-contributor-responsibilities) to understand your role.

---
## Our Charter 📜

> The **[Async API Charter](./CHARTER.md)** outlines the structure, responsibilities, and procedures for contributions and oversight of the AsyncAPI open project. <br>

Reviewer: ```suggestion
The **[AsyncAPI Charter](./CHARTER.md)** outlines the structure, responsibilities, and procedures for contributions and oversight of the AsyncAPI open project. <br>
```

 #### Key sections include:

  - **Mission and Scope**: Defines the project's mission to provide open-source, vendor-neutral API specifications and tools.
  - **Technical Steering Committee (TSC)**: Describes the TSC's role in project oversight, member composition, and voting procedures.
  - **Committers and Contributors**: Details the roles and responsibilities of contributors and committers.
  - **TSC Voting**: Explains the voting process for decision-making.
  - **Amendments**: Procedures for amending the charter.
  - **Intellectual Property Policy**: Governs the handling of intellectual property.
  - **Compliance with Policies**: Ensures adherence to LF Projects policies.
  - **General Rules and Operations**: Sets professional and ethical standards.
  - **Community Assets**: Manages trademarks and assets.

---

> ## Community Governance 🏛️

Reviewer: Remove this section we are having a governance board soon

Our community is steered by a transparent governance. Learn about our  **[Governance](./GOVERNANCE.md)** and decision-making process.

   - ### Executive Director
       Our Current Executive Director is **Lukasz Gornicki**. 📧 [Email](mailto:lpgornicki@gmail.com) | 💬 AsyncAPI Slack: Lukasz Gornicki (User ID: UD698Q5LM) | 🐙 [GitHub](https://github.com/derberg) | 🐦 [Twitter](https://twitter.com/derberq)

   - ### Past Directors
      **Fran Mendez** 📧 [Email](mailto:fmvilas@gmail.com) | 💬 AsyncAPI Slack: Fran Méndez (User ID: U34F2JRRS) | 🐙 [GitHub](https://github.com/fmvilas) | 🐘 [Mastodon](https://social.fmvilas.me/@fran)

---

> ## Async API Technical Steering Committee (TSC) 👥

Reviewer 3: Like this @Sonalilipsa17375 
```suggestion
## Becoming a TSC or an AsyncAPI Ambassador 👥
```

> ## Async API Technical Steering Committee (TSC) 👥

Reviewer: Make the TSC and Ambassador sections into one. Maybe rename to "Becoming a TSC or an AsyncAPI Ambassador"

- ### How to Be a part of TSC 🤝
  - **Become a Maintainer**: Initiate, donate, or support projects in the AsyncAPI GitHub organization.
  - **Join the TSC**: Maintainers automatically become TSC members without voting.

    If you prefer video, watch our **[YouTube video on how to become a TSC member](https://www.youtube.com/watch?v=uG_aLF9Z1F0)**. <br>

    #### Find out more about **[TSC Membership](./TSC_MEMBERSHIP.md)**. <br>

- ### **Meet Async API's current **[TSC](https://www.asyncapi.com/community/tsc)** members**.

---

## AsyncAPI Ambassador Program 🌟

Welcome to the AsyncAPI Ambassador Program! This program is designed to bring AsyncAPI closer to both current users and potential new users. 

### What is the AsyncAPI Ambassador Program? ✨

The AsyncAPI Ambassador Program provides tools and resources to:
- Promote user cooperation and share necessary tools.
- Engage with the project as a contributor, blogger, speaker, etc.
- Speak at community events or write technical content.
- Nurture community growth and spread knowledge about the project.

### Explore more at [Async API Ambassador Program](./AMBASSADOR_ORGANIZATION.md)
---

## Community Discussions 💬

> AsyncAPI Community uses **Slack as the source of communication**. It helps us stay closer to each other and to know each other better through discussions.

Reviewer: Actually, Slack isn't the only form of communication. Maybe it can be something to add in the beginning of the page under welcome.

>  For more details, refer to our **[Slack Etiquette](./slack-etiquette.md)** guidelines.

Reviewer: This should also go under contributing or getting started.

  ### Need Help? 🤝

>   If you have any questions or need guidance, feel free to reach out through our **[Community Discussions](https://asyncapi.com/slack-invite)**.

Reviewer 3: I think this is what @thulieblack meant.
```suggestion
  If you have any questions or need guidance, feel free to reach out through our **[GitHub Discussions page](https://github.com/orgs/asyncapi/discussions)**.
```

>   If you have any questions or need guidance, feel free to reach out through our **[Community Discussions](https://asyncapi.com/slack-invite)**.

Reviewer: Github discussions link shouldn't take you to slack

 
 [![AsyncAPI Logo](./assets/github-repobanner-community.png)](https://www.asyncapi.com)

# 📘 AsyncAPI Community Guide

Welcome to the AsyncAPI Community! 🚀 We believe in the power of open-source collaboration and appreciate your interest in contributing. Whether you're improving the documentation, fixing bugs, or adding new features, your contributions help make AsyncAPI better for everyone. 💙

### 🔖 Code of Conduct
Before you begin, please **go through our [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md)** to ensure a respectful and welcoming community. Find more about contributing to our Docs at **[AsyncAPI Docs Communtiy](https://www.asyncapi.com/docs/community)**.

---

## 🛠 How to Contribute to Our docs

We believe that **code is cool**, but **great docs make all the difference**!

- Read the [AsyncAPI Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).
- Go through the [AsyncAPI Slack etiquette](https://github.com/asyncapi/community/blob/master/slack-etiquette.md).

> - Join our [AsyncAPI Docs Community](https://www.asyncapi.com/docs/community).

Reviewer: We don't have a docs community

Author: Thanks for the suggestions, I have incorporated this change.

- Add the AsyncAPI calendar from the [AsyncAPI events](https://www.asyncapi.com/community/events) page to stay updated on meetings and events.

> - Review the list of [technical writer contributor](https://www.asyncapi.com/docs/community/onboarding-guide/technical-writer-contributor-responsibilities) responsibilities to understand your role.

Reviewer: Only one link to TW onboarding is needed here. Community repo is not only for docs

Author: Yes updated it.

- Familiarize yourself with [prerequisite knowledge topics](https://www.asyncapi.com/docs/community/onboarding-guide/prerequisite-knowledge) required for contributing.
- Check out the work-in-progress [AsyncAPI Style Guide](https://github.com/asyncapi/community/pulls?q=is%3Apr+is%3Aopen+style+guide) to ensure consistency in documentation.
- Read all documentation under the following content buckets:
    - Concepts
    - Tutorials (take time to go through each tutorial)
    - Reference
- Set up your local environment following the [AsyncAPI git workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md).

> - Introduce yourself in AsyncAPI Slack in the **#01_introductions channel** and the #13_docs channel. Ask docs-related questions in #13_docs.

Reviewer: This is not needed

> - Request a good first docs issue in the **#13_docs** [Slack channel](https://asyncapi.slack.com/ssb/redirect).

Reviewer: Same here, this is a community repo so it should reflect important community docs not for tech writers

Author: I have modified it, pls share your suggestions. @thulieblack
